examples/fastapi/chat-with-github-api/requirements.txt

```diff
@@ -8,4 +8,6 @@ sqlalchemy
 llama-index-core
 llama-index-readers-github
 llama_index
-humanize
\ No newline at end of file
+humanize
+solara==1.29.1
+openai
```

Comment:
provide specific versions so the app doesn't break with upgrades

Reply:
[added versions](https://github.com/ploomber/doc/pull/170/files#diff-21fe5dc251f728d09d24529cc8556e342bffd8e35528ebac4ee072c8a0334b3cR2-R13)